docs/adding_software/debugging_failed_builds.md

```diff
@@ -45,36 +45,60 @@ Simply run the EESSI container (`eessi_container.sh`), which should be in the ro
 ```
 ./eessi_container.sh --access rw
 ```
+
+If you want to install NVIDIA GPU software, make sure to also add the `--nvidia all` argument, to insure that your GPU drivers get mounted inside the container:
+```
+./eessi_container.sh --access rw --nvidia all
+```
+
 !!! Note
     You may have to press enter to clearly see the prompt as some messages
     beginning with `CernVM-FS: ` have been printed after the first prompt
     `Apptainer> ` was shown.
 
-If you want to debug an issue for which a lot of dependencies need to be build first, you may want to start the container with the `--save DIR/TGZ` and flag (check `./eessi_container.sh --help`). This saves the temporary directory (which we will use as working and installation directory later in this instruction) in order to be able to resume later with the same temporary directory. E.g.
 
+### Starting a shell in the EESSI container - more efficient approach
```

Comment:
Maybe "more efficient approach for multiple debugging sessions" or something like that?

Reply:
Done!